python-namespace/globals_func.py

```diff
@@ -0,0 +1,5 @@
+globals()["message"] = "Welcome to Real Python!"
+
+# print(message)
+# globals()["message"] = "Hello, World!"
+# print(message)
```

Comment:
@lpozo I assume you commented these out to satisfy the linter checks? In the future, you can use the special [`# noqa`](https://stackoverflow.com/questions/45346575/what-does-noqa-mean-in-python-comments) comment to suppress the linter.

Reply:
Apparently this doesn't work:

<img width="841" alt="image" src="https://github.com/user-attachments/assets/a3b0acfe-1172-4a9d-8e91-1c521c2bfcce" />